testsuite/integration-arquillian/tests/base/testsuites/login-suite

```diff
@@ -0,0 +1,15 @@
+AdminConsoleWhoAmILocaleTest
+AppInitiatedActionUpdateEmailWithVerificationTest
+AuthorizationCodeTest
+IdpInitiatedLoginTest
+KcOidcFirstBrokerLoginTest
+KcOidcMultipleTabsBrokerTest
+LoginPageTest
+RefreshTokenTest
+RegisterWithUserProfileTest
+RequiredActionUpdateProfileTest
+RequiredActionUpdateProfileWithUserProfileTest
+SAMLServletAdapterTest
+ThemeSelectorTest
+VerifyProfileTest
+X509BrowserLoginTest
```

Comment:
Some of these don't really focus on login pages, I suggest something like this to start:

```suggestion
org.keycloak.testsuite.actions
org.keycloak.testsuite.forms
KcOidcBrokerLoginHintTest
KcOidcFirstBrokerLoginTest
KcOidcMultipleTabsBrokerTest
LoginPageTest
```